src/components/Common/EllipsisText/EllipsisText.style.ts

```diff
@@ -0,0 +1,22 @@
+import styled, { css } from 'styled-components';
+
+interface StyledButtonProps {
+  $width: string;
+  $fontSize: string;
+  $fontWeight: number;
+  $lineClamp: number;
+}
+
+export const BaseEllipsisText = styled.span<StyledButtonProps>`
+  ${({ $width, $fontSize, $fontWeight, $lineClamp }) => css`
+    display: -webkit-box;
+    width: ${$width};
+    overflow: hidden;
+    font-size: ${$fontSize};
+    font-weight: ${$fontWeight};
+    text-overflow: ellipsis;
+    word-wrap: keep-all;
```

Comment:
word-break: keep-all; 을 의도하신 게 맞을까요?

만약 word-break라면 keep-all의 경우 띄어쓰기가 없는 긴 글은 ...이 적용이 안 되는 문제가 있네요!
수정한다면 `word-wrap: break-word;`이 좋을 것 같습니다.

Reply:
```suggestion
    word-break: keep-all;
    word-wrap: break-word;
```

수영님 의견대로 위와 같이 변경하면 될 거 같습니다!

Reply:
건호님 PR과 통일해서 이렇게 변경해보겠습니다!
다들 좋은 의견 감사드립니다 😁

```tsx
word-break: keep-all;
word-wrap: break-word;
white-space: pre-wrap;
```